huaweicloud/services/vpc/resource_huaweicloud_vpc_subnet.go

```diff
@@ -171,10 +171,11 @@ func ResourceVpcSubnetV1() *schema.Resource {
 				Computed: true,
 			},
 			"availability_zone": {
-				Type:     schema.TypeString,
-				Optional: true,
-				ForceNew: true,
-				Computed: true,
+				Type:        schema.TypeString,
```

Comment:
It would be appreciated if you also moved this parameter in front of the other optional parameters.

Reply:
fixed.